README.md

```diff
@@ -108,7 +56,9 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://game-of-thrones-iota.vercel.app/)
+
+- [Explanation video](https://www.loom.com/share/b9afc80c2a7c47ec82b2193ae078ec9b)
 
```

Comment:
- [ ] I recommend you to be more specific in your code. Instead of show all codes, you should highlight a piece of the code you find interesting or you are proud of making.

Reply:
done